values.tf

```diff
@@ -0,0 +1,37 @@
+locals {
+  values_default = yamlencode({
+    image = {
+      tag = var.grafana_release
```

Comment:
maybe we should leave it to the chart defaults

Reply:
The ability to pass specific tag is on the current version of the addon, so I would leave it there to retain compatibility. How this works internally in the Helm chart is that if the tag is empty, the default value is taken from `.Chart.appVersion` meaning the default version is taken from the Helm chart itself.

Reply:
removed